stix_shifter/stix_transmission/src/modules/guardium/guardiumapiclient.py

```diff
@@ -0,0 +1,448 @@
+from ..utils.RestApiClient import RestApiClient
+from ..utils.RestApiClient import ResponseWrapper
+from requests.models import Response
+import urllib.parse
+import logging
+import sys
+import json
+import datetime
+import os
+import base64
+
+
+class APIClient():
+
+    # API METHODS
+
+    # These methods are used to call Ariel's API methods through http requests.
```

Comment:
This should reference Guardium rather than Ariel.

Reply:
fixed.